examples/api_test/test_ssspapi.cu

```diff
@@ -0,0 +1,128 @@
+/*
+ * @brief Subgraph matching test for shared library advanced interface
+ * @file test_smapi.cu
+ */
+
+ #include <stdio.h>
+
+ #include <thrust/copy.h>
+ #include <thrust/host_vector.h>
+ #include <thrust/device_vector.h>
+ 
+ #include <gunrock/gunrock.h>
```

Comment:
Do you need the `gunrock.h` include here?

Reply:
Def not, removed in latest commit.